libs/sdk-ui-loaders/api/sdk-ui-loaders.api.md

```diff
@@ -78,33 +78,33 @@ export interface IDashboardLoader {
     withFilterContext(filterContextRef: ObjRef): IDashboardLoader;
 }
 
-// @alpha
+// @public
 export interface IDashboardLoadOptions extends IDashboardBasePropsForLoader {
     adaptiveLoadOptions?: AdaptiveLoadOptions;
     clientWorkspace?: IClientWorkspaceIdentifiers;
     extraPlugins?: IEmbeddedPlugin | IEmbeddedPlugin[];
     loadingMode?: "adaptive" | "staticOnly";
 }
 
-// @alpha (undocumented)
+// @public (undocumented)
```

Comment:
Public thing without docs, please check it.

Reply:
There are actually several reported in this file, can you please check them?

Reply:
fixed